src/utils/api.ts

```diff
@@ -133,10 +133,28 @@ const api = {
     },
   },
   activePositions: {
-    nodeInfo: async () => {
-      return await fetch.get<NodeInfo[]>(
-        `https://thornode.ninerealms.com/thorchain/nodes`
-      );
+    nodeInfo: (address: string): Promise<number> => {
```

Comment:
I prefer using .then() and .catch() because it provides granular control over error handling and aligns with the standard I’ve applied across the codebase, especially in functions with more complex logic and varied error handling. While this function may look simple, maintaining consistency ensures clarity and robustness throughout the project